tidb-server/main.go

```diff
@@ -89,163 +139,74 @@ var (
 		})
 )
 
-func main() {
-	tidb.RegisterLocalStore("boltdb", boltdb.Driver{})
-	tidb.RegisterStore("tikv", tikv.Driver{})
-	tidb.RegisterStore("mocktikv", tikv.MockDriver{})
-
-	runtime.GOMAXPROCS(runtime.NumCPU())
+var (
+	cfg     *config.Config
```

Comment:
This is actually a pointer to gloablConfig?

Reply:
Yes.